nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TransformXml.java

```diff
@@ -76,25 +84,59 @@
             .name("XSLT file name")
             .description("Provides the name (including full path) of the XSLT file to apply to the flowfile XML content.")
             .required(true)
+            .expressionLanguageSupported(true)
             .addValidator(StandardValidators.FILE_EXISTS_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor INDENT_OUTPUT = new PropertyDescriptor.Builder()
+            .name("indent-output")
+            .displayName("Indent")
+            .description("Whether or not to indent the output.")
+            .required(true)
+            .defaultValue("true")
+            .allowableValues("true", "false")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor CACHE_SIZE = new PropertyDescriptor.Builder()
+            .name("cache-size")
+            .displayName("Cache size")
+            .description("Maximum size of the stylesheet cache.")
+            .required(true)
+            .defaultValue("100")
+            .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor CACHE_DURATION = new PropertyDescriptor.Builder()
+            .name("cache-duration")
+            .displayName("Cache duration")
+            .description("How long to keep stylesheets in the cache.")
```

Comment:
This property is misleading. What I initially thought it to be was how long the stylesheets will exist in the cache (deleted after 60s). What it actually is, is an expiration time where after 60s of it not being used it will be removed.